packages/server/src/db/index.js

```diff
@@ -433,15 +436,15 @@ async function getTotalInterestedGrantsByAgencies(agencyId) {
     const agencies = await getAgencyTree(agencyId);
     const rows = await knex(TABLES.grants_interested)
         .select(`${TABLES.grants_interested}.agency_id`, `${TABLES.agencies}.name`, `${TABLES.agencies}.abbreviation`,
-            knex.raw('SUM(CASE WHEN is_rejection = TRUE THEN 1 ELSE 0 END) rejections'),
-            knex.raw('SUM(CASE WHEN is_rejection = FALSE THEN 1 ELSE 0 END) interested'),
+            knex.raw('SUM(CASE WHEN status_code = "Rejected" THEN 1 ELSE 0 END) rejections'),
+            knex.raw('SUM(CASE WHEN status_code = "Interested" THEN 1 ELSE 0 END) interested'),
```

Comment:
@mhuang-usdr it looks like we calculate total rejections / total interested as well as total rejected grant money / total interested grant money - how would the new `Result` grants be calculated in this?

Reply:
Let's leave Results out of the totals for now - that needs to be thought through more first.

Reply:
I'm assuming this is related to the `Total Interested Grants by Agencies` dashboard, but we don't have a good plan to surface Results there yet. So I'd saw let's leave it out for now.